CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ endif()
 
 project (cil LANGUAGES CXX)
 
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.27)
+  cmake_policy(SET CMP0148 OLD)
```

Comment:
technically we should be using `NEW` and updating `FindPython` usage everywhere

Reply:
yes, but in that case we don't set the policy, we just use the new functionality.